shapepipe/modules/mask_package/mask.py

```diff
@@ -486,25 +535,28 @@ def mask_border(self, width=100, flag_value=4):
 
         return flag
 
-    def mask_messier(self, cat_path, size_plus=0.1, flag_value=8):
-        """Mask Messier.
+    def mask_dso(self, cat_path, size_plus=0.1, flag_value=8, typ='Messier'):
+        """Mask DSO.
 
-        Create a circular patch for Messier objects.
+        Create a circular patch for deep-sky objects (DSOs), e.g.
+        Messier or NGC objects.
 
         Parameters
         ----------
         cat_path : str
-            Path to the Messier catalogue
+            Path to the deep-sky catalogue
         size_plus : float
             Increase the size of the mask by this factor
             (e.g. ``0.1`` means 10%)
         flag_value : int
             Value of the flag, some power of 2
+        typ : str, optional
```

Comment:
I think `obj_type` would be a better variable name. Also if the variable only takes one of two possible string values you can do the following (according to the [Numpy doc style guide](https://numpydoc.readthedocs.io/en/latest/format.html)) e.g.:

```python
obj_type : {'Messier', 'NGO'}, optional
```

This implies rather than accepting a general `str` object it has to be one of the strings specified. Also this would be a good option to test for.